docs/components/control/redirect-to-signin.mdx

```diff
@@ -11,7 +11,38 @@ The `<RedirectToSignIn />` component will navigate to the sign in URL which has
 
 <InjectKeys>
 
-<Tabs type="framework" items={["Next.js", "React", "Remix"]}>
+<Tabs type="framework" items={["Next.js App", "Next.js Pages", "React", "Remix"]}>
```

Comment:
So "Next.js App" and "Next.js Pages" are not going to sync throughout the docs. It needs to say "Next.js" and then within the Next.js `<Tab>`, you can add `<CodeBlockTabs options={["App Router", "Pages Router"]} />`

```suggestion
<Tabs items={["Next.js", "React", "Remix"]}>
```